components/admin_console/admin_definition.jsx

```diff
@@ -1602,26 +1844,29 @@ const AdminDefinition = {
                         help_text_default: 'When false, only System Administrators can create teams.',
                         permissions_mapping_name: 'enableTeamCreation',
                         isHidden: it.licensed,
+                        isDisabled: it.not(it.userHasWritePermissionOnResource('site')),
                     },
                     {
                         type: Constants.SettingsTypes.TYPE_NUMBER,
                         key: 'TeamSettings.MaxUsersPerTeam',
                         label: t('admin.team.maxUsersTitle'),
                         label_default: 'Max Users Per Team:',
                         help_text: t('admin.team.maxUsersDescription'),
-                        help_text_default: 'Maximum total number of users per team, including both active and inactive users.',
+                        help_text_default: 'Maximum total number of users per team, including all active and inactive users.',
                         placeholder: t('admin.team.maxUsersExample'),
                         placeholder_default: 'E.g.: "25"',
+                        isDisabled: it.not(it.userHasWritePermissionOnResource('site')),
                     },
                     {
                         type: Constants.SettingsTypes.TYPE_NUMBER,
                         key: 'TeamSettings.MaxChannelsPerTeam',
                         label: t('admin.team.maxChannelsTitle'),
                         label_default: 'Max Channels Per Team:',
                         help_text: t('admin.team.maxChannelsDescription'),
-                        help_text_default: 'Maximum total number of channels per team, including both active and archived channels.',
+                        help_text_default: 'Maximum total number of channels per team, including all active and archived channels.',
```

Comment:
are these translation changes intentional? if so en.json will also need to be updated

Reply:
I have no idea how that happened. Fixed.